frontend/src/pages/budgetLines/list/BudgetLineItemList.jsx

```diff
@@ -151,14 +154,16 @@ const BudgetLineItemList = () => {
                         const feeRate = calculateProcShopFeePercentage(budgetLine, procShopMap[budgetLine.id] || 0);
                         return [
                             budgetLine.id,
-                            budgetLine.agreement?.project?.title ?? "TBD",
-                            budgetLine.agreement?.name ?? "TBD",
+                            budgetLine.agreement?.project?.title ?? NO_DATA,
+                            budgetLine.agreement?.name ?? NO_DATA,
                             budgetLinesDataMap[budgetLine.id]?.service_component_name,
+                            budgetLine.agreement?.agreement_type ?? NO_DATA,
                             budgetLine.line_description,
                             formatDateNeeded(budgetLine?.date_needed ?? ""),
                             budgetLine.fiscal_year,
-                            budgetLine.can?.display_name ?? "TBD",
+                            budgetLine.can?.display_name ?? NO_DATA,
```

Comment:
@fpigeonjr @weimiao67 For what it's worth, I tried also adding in the BL's CAN's portfolio abbreviation with `budgetLine.can?.portfolio.abbreviation` but it didn't work, although it seemingly should have since the CAN imports the full Portfolio type.  🤷